Sources/Logger/Logger.swift

```diff
@@ -2,6 +2,8 @@
 import Utils
 
 public struct Logger: Sendable {
+    public nonisolated(unsafe) static var level: OSLogType = .fault
```

Comment:
this should be set once and never again, so I wasn't so worried about thread-safety here